.eslintrc.json

```diff
@@ -21,7 +21,7 @@
         "prettier/prettier": 2,
         "import/no-default-export": 2,
         "interface-over-type-literal": 0,
-        "@typescript-eslint/prefer-interface": 0,
+        "@typescript-eslint/consistent-type-definitions": 0,
```

Comment:
Release `typescript-eslint@2.0.0` removed deprecated rule `prefer-interface`. This rule was replaced by `consistent-type-definitions`.

https://github.com/typescript-eslint/typescript-eslint/releases/tag/v2.0.0